core/server/proxy/src/main/java/alluxio/proxy/s3/ListBucketResult.java

```diff
@@ -218,27 +212,27 @@ public String getNextMarker() {
    * @return the common prefixes
    */
   @JacksonXmlProperty(localName = "CommonPrefixes")
-  public CommonPrefixes getCommonPrefixes() {
+  @JacksonXmlElementWrapper(useWrapping = false)
```

Comment:
why add `useWrapping = false` here?

Reply:
Refer to the annotations of `mContents`.

```
  /**
   * @return the list of contents
   */
  @JacksonXmlProperty(localName = "Contents")
  @JacksonXmlElementWrapper(useWrapping = false)
  public List<Content> getContents() {
    return mContents;
  }
```

Through testing, found that `useWrapping = false` is not necessary and can be deleted.